CHANGELOG.md

```diff
@@ -8,25 +8,31 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Added
 
-- Added `--ttnv3.no-session` and `--ttnv3.delete-source-device` flags
-
 ### Changed
 
 ### Fixed
 
-- `app-id` flag/variable is no longer needed when calling `application` command with `ttnv2` and `ttnv3` sources
-- `application` command no longer panics when called with a different `app-id` argument than set with flag/variable
+- Using `--ttnv3.no-source` flag will no longer crash.
+- End Device Identifiers are now exported from all available servers.
 
 ### Removed
 
 ### Security
 
 ## [v0.9.0] (2022-09-13)
 
-- `ttnv3` source to migrate devices between TTS instances
-
 ### Added
 
+- `ttnv3` source to migrate devices between TTS instances.
```

Comment:
The formatting here is a bit anachronistic.

Specifically, there never was an official `v0.9.0` release, only some releases candidates, so having a date for the `v0.9.0` release is a bit weird.

Secondarily, a released version should not have these empty sections like `Changed` (we strip them on release).

Finally, some of these changes that are mentioned here come from https://github.com/TheThingsNetwork/lorawan-stack-migrate/pull/65 , not this PR, and should be added from that one instead.

Reply:
> Finally, some of these changes that are mentioned here come from https://github.com/TheThingsNetwork/lorawan-stack-migrate/pull/65 , not this PR, and should be added from that one instead.

I wanted to move some of the changes into this PR, as they are not tied to cli improvements, like tidying up the changelog. Changes related to https://github.com/TheThingsNetwork/lorawan-stack-migrate/pull/65 were not carried in here.